container/docker/handler.go

```diff
@@ -294,32 +294,8 @@ func (self *dockerContainerHandler) GetStats() (*info.ContainerStats, error) {
 }
 
 func (self *dockerContainerHandler) ListContainers(listType container.ListType) ([]info.ContainerReference, error) {
-	if self.name != "/docker" {
```

Comment:
Maybe we are saying the same thing. Or are you saying this code is still needed? If so, do you know where the `dockerContainerHandler` named *only* `/docker` is created? Because only then will `ListContainers` actually do something.

In my tests, with this function stubbed, everything also still works on a non-systemd machine and also see Docker containers with the `/docker` cgroup eg.

```
clever_wright (/docker/64f26ce9d60219c5d730f31abeabc39c47fe7aa282343c3a3a8c820a80db5b70)
```